content/vendor/guides/using-ecr-private-images.md

```diff
@@ -0,0 +1,319 @@
+---
+date: "2020-09-24T00:00:00Z"
+lastmod: "2020-09-24T00:00:00Z"
+title: "Using ECR for Private Images"
+weight: "1005"
+---
+
+KOTS supports working with private images stored in Amazon's Elastic Container Registry (ECR).
+
+## Objective
+
+The purpose of this guide is to walk you through a hello-world example on how to configure Replicated to pull images from a private registry in Amazon's Elastic Container Registry (ECR).
+
+## Prerequisites & Assumptions
+
+This power-user's guide assumes you have completed the [standard quickstart](https://kots.io/vendor/guides/quickstart/) or the [CLI quickstart](https://kots.io/vendor/guides/cli-quickstart/) guides as this guide is a continuation of those guides.
+As with the previous guides, we will also need a VM to install the application with the following minimum requirements:
+
+* Ubuntu 18.04
+* At least 8 GB of RAM
+* 4 CPU cores
+* At least 100GB of disk space
+
+In this guide we are going to focus on the difference between using a public image versus a private image in Replicated.
+To do this, we'll pull the public ngnix container and then push it to a private repository in ECR.
+
+This means we'll need:
+
+* An ECR Repository
+* An AWS Account to use with Docker to pull and push the public nginx image to the ECR repository
+* Docker
+* The AWS CLI
+
+Later in the guide we'll configure Replicated to pull from the ECR repository using a read-only account.
+To do this we'll need to make sure the above AWS account can also create this user.
+
+## Overview
+
+The guide is divided into the following steps:
+
+ 1. [Set Up Testing Environment](#1-set-up-testing-environment) 
+ 
+ 2. [Configure Private Registries in Replicated](#2-configure-private-registries-in-replicated)
+
+ 3. [Update Definition Files](#3-update-definition-files)
+
+ 4. [Install the New Version](#4-install-the-new-version)
+
+## 1. Set Up Testing Environment
+
+For this guide, we are simply going to use the default nginx deployment to create our application and then update it to pull the same container from a private repository in ECR and note the differences.
```

Comment:
Also, if this can be a standalone guide, should we provide the default *nignx YAML, or reference it? Some people may choose not to follow the guides from start to finish.

Reply:
The goal wasn't to make it a stand alone guide, per se. The reader should be familiar with the default nginx deployment for the initial KOTS release.